training/requirements.txt

```diff
@@ -6,5 +6,4 @@ jiwer
 evaluate>=0.4.1
 wandb
 tensorboard
-tensorflow-cpu
```

Comment:
we don't need tensorflow no?

Reply:
Ah yes we can remove! `tensorflow-cpu` is required for Flax's tensorboard logs, but should be ok for PT 👍